ckanext/harvest/commands/harvester.py

```diff
@@ -455,6 +455,10 @@ def run_test_harvest(self):
                 job_dict = jobs[0]
         job_obj = HarvestJob.get(job_dict['id'])
 
+        if self.args[0] == 'run_test':
```

Comment:
I'm not sure if we need this since we're already in the `run_test` method.

Reply:
Thats true, I'll correct it